programs/whirlpool/src/instructions/open_bundled_position.rs

```diff
@@ -69,11 +71,19 @@ pub fn handler(
 
     position_bundle.open_bundled_position(bundle_index)?;
 
+    // Resolve one-sided positions (if one bound is auto-derived from current price)
```

Comment:
nit: is this comment really adding anything? imo the function name and vars are nicely named 🙂